docs/contributing.rst

```diff
@@ -100,7 +100,7 @@ HTTPS. Its three purposes are to be:
    for publishing on any Python package index
 
 
-Currently, twine has two principal functions: uploading new packages
+Currently, twine has two principle functions: uploading new packages
```

Comment:
Not to nitpick, but I believe this change is wrong: https://english.stackexchange.com/questions/71970/principle-or-principal

Reply:
Interesting. I always learned that "principal" referred to a person. Happy to create another PR to revert the change -- let me know. @di

Reply:
What if we just remove the point of contention altogether and say "Currently, twine has two primary functions:"? Alternatively, let's maybe find a better way to say that twine's main responsibilities are uploading and registering packages.